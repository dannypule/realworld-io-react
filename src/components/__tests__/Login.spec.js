test("show match snapshot", () => {
  expect([2, 2]).toMatchSnapshot();
});
