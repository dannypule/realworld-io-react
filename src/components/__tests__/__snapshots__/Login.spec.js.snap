// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`show match snapshot 1`] = `
Array [
  2,
  2,
]
`;
